platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -66,6 +68,7 @@ const EVENTS = {
   MEASUREMENT_REMOVED: 'event::measurement_removed',
   MEASUREMENTS_CLEARED: 'event::measurements_cleared',
   JUMP_TO_MEASUREMENT: 'event:jump_to_measurement',
+  MEASUREMENT_SELECTED_CHANGED: 'event:measurement_selected_changed',
```

Comment:
we are using `updated` in other events. Can we stick to updated for this one too? `MEASUREMENT_SELECTED_UPDATED`

Reply:
Yes, that makes sense.